### DISCUSSION TITLE
Is it possible to select localfile url text and send it to Ob?

### DISCUSSION THREAD

**Discussion by etjec4** [coddingtonbear/obsidian-web#146] [Q&A]
it doesn't seems to work.

**Comment by coddingtonbear** [comment]
I'm afraid I don't understand what your question is -- could you elaborate on what you mean?

**Reply by etjec4** [reply]
I have a local file on my disk, 
I open it in chrome, file address like file:///D:/Users/readme.txt
Now I'd like send some notes to Obsidian using obsidian-web
Then it said 

Error
Could not send content to Obsidian: (Error Code 40400) Not Found


:)
should be clear.
